### DISCUSSION TITLE
ComputerCraft + HBM

### DISCUSSION THREAD

**Discussion by solundev** [HbmMods/Hbm-s-Nuclear-Tech-GIT#543] [Q&A]
Is there a way to create a reactor monitor with ComputerCraft? if so, how?

**Comment by Shibva** [comment]
actually it seems like this was added recently and im glad its a thing, this could be the start of some very interesting stuff

**Comment by BallOfEnergy1** [comment]
Hey, late reply, but CC (ComputerCraft) Does not have integration with HBM NTM, however, OC (OpenComputers) does have integration with NTM.

**Comment by m-doescode** [comment]
It is a little bit of a shameful plug but I did make a super simple ComputerCraft-HBM integration mod a while back and I just published it to GitHub.

If you're interested, it's here: https://github.com/m-doescode/cchbm
